test/10-verify.spec.js

```diff
@@ -808,7 +808,7 @@ for(const [version, mockCredential] of versionedCredentials) {
           .contain('"issuer" must be a URI');
       });
 
-      it('should reject credentialStatus that is not a URI', () => {
+      it('should reject credentialStatus id that is not a URI', () => {
```

Comment:
the spec says URL actually. Also, only one id is allowed.

Reply:
I don't know what needs changing here.  Please do whatever is needed.  If only one status is allowed, the check code should check that, and tests should be written to fail on multiple statuses.